src/transforms/transform-pull-request.ts

```diff
@@ -198,15 +200,21 @@ export const transformPullRequest = (_jiraHost: string, pullRequest: pullRequest
 	return {
 		author: getJiraAuthor(pullRequest.author),
 		commentCount: pullRequest.comments.totalCount || 0,
-		destinationBranch: pullRequest.baseRef?.name || "",
-		destinationBranchUrl: `https://github.com/${pullRequest.baseRef?.repository?.owner?.login}/${pullRequest.baseRef?.repository?.name}/tree/${pullRequest.baseRef?.name}`,
+		destinationBranch: pullRequest.baseRefName || "",
+		destinationBranchUrl: `https://github.com/${repository.owner?.login}/${repository.name}/tree/${pullRequest.baseRefName}`,
 		displayId: `#${pullRequest.number}`,
 		id: pullRequest.number,
 		issueKeys,
 		lastUpdate: pullRequest.updatedAt,
 		reviewers: mapReviews(pullRequest.reviews?.nodes, pullRequest.reviewRequests?.nodes),
-		sourceBranch: pullRequest.headRef?.name || "",
-		sourceBranchUrl: `https://github.com/${pullRequest.headRef?.repository?.owner?.login}/${pullRequest.headRef?.repository?.name}/tree/${pullRequest.headRef?.name}`,
+		sourceBranch: pullRequest.headRefName,
+		...(
+			pullRequest.headRef
+				? {
+					sourceBranchUrl: `https://github.com/${pullRequest.headRef?.repository?.owner?.login}/${pullRequest.headRef?.repository?.name}/tree/${pullRequest.headRef?.name}`
```

Comment:
Need a bit of thinking for GHE as it won't be github.com

Reply:
I think we can use headRef.repository.url to replace the need for `github.com`

Reply:
Nice find @gxueatlassian , but I think that exactly the same logic that was before. We probably should raise this as a bug and not consider it as a blocker, wdyt?